include/wx/xrc/xh_choicbk.h

```diff
@@ -27,6 +27,8 @@ class WXDLLIMPEXP_XRC wxChoicebookXmlHandler : public wxXmlResourceHandler
 private:
     bool m_isInside;
     wxChoicebook *m_choicebook;
+    wxVector<wxBitmapBundle> m_bookImages;
+    wxVector<size_t> m_bookImagesIdx;
```

Comment:
It would be nice to explain somewhere what does this vector hold. Of course, the not very nice thing is that it's used in several places and we don't want to repeat the same description in all of them, so ideal would be to add some `wxBookCtrlXmlResourceHandlerBase` class and put this field and the above one in into it and write this comment just once there.